### PR TITLE
Enable api_v3_syntaxConformaceTest::testInvalidID_delete

### PR DESCRIPTION
**Pull request by seamuslee001**

### CODE REVIEW

File: api/v3/SystemLog.php [hunk 1]
```diff
@@ -39,7 +39,7 @@
  * @return array
  */
 function civicrm_api3_system_log_delete($params) {
-  return _civicrm_api3_basic_get(_civicrm_api3_get_BAO(__FUNCTION__), $params, TRUE, 'SystemLog');
+  return _civicrm_api3_basic_delete(_civicrm_api3_get_BAO(__FUNCTION__), $params);
```
Comment (colemanw) [inline api/v3/SystemLog.php]: This is really weird!

**Review by colemanw — COMMENTED** [review]
Congrats on getting this test passing! What's up with the SystemLog api?

File: api/v3/SystemLog.php (hunk 1 above)
Comment (seamuslee001) [inline api/v3/SystemLog.php]: ah now seeing comment in relevance not just from MM channel can see your point. Yeh i dunno it smells of copy and paste to me

**Comment by seamuslee001** [comment]
@colemanw I can only assume that systemlog.delete was a copy and paste error or never really implemented the delete function

File: api/v3/SystemLog.php (hunk 1 above)
Comment (colemanw) [inline api/v3/SystemLog.php]: Any idea @eileenmcnaughton ?

Comment (eileenmcnaughton) [inline api/v3/SystemLog.php]: I can't imagine what sort of idiot would have made such a stupid error.....

Comment (eileenmcnaughton) [inline api/v3/SystemLog.php]: (actually I can :-))

**Comment by seamuslee001** [comment]
@colemanw you think we can merge this in given it appears the SystemLog was a human error now being fixed?

**Comment by colemanw** [comment]
Done :)
Is there a jira ticket (ok if not, just want to make sure we close it if so)

**Pull request merged**